src/headers/_mod.rs

```diff
@@ -466,6 +498,12 @@ enum Language {
     CSharp,
 }
 
+pub enum NamingConvention {
+    Default,
+    Suffix(String),
+    Prefix(String),
```

Comment:
🤔 what do you think of:

```suggestion
    Default,
    Custom(fn(&str) -> String),
```

it would be more general, and would still allow `suffix` and `prefix` with things like: `Custom(|s| format!("{s}_t"))`.

___

Otherwise we could handle them all:

```suggestion
    Default,
    Suffix(String),
    Prefix(String),
    Custom(fn(&str) -> String),
```

Reply:
Handling them all is nice because we could handle something like :
```rust
#[ffi_export(prefix="PFX")]
struct Foo {...}
```